crates/wasi-nn/src/lib.rs

```diff
@@ -84,6 +84,12 @@ impl fmt::Debug for Tensor {
     }
 }
 
+impl PartialEq for Tensor {
+    fn eq(&self, other: &Self) -> bool {
+        self.dimensions == other.dimensions && self.ty == other.ty && self.data == other.data
+    }
+}
+
```

Comment:
Probably should just `#[derive(PartialEq)]` since it's equivalent to this I think.

Reply:
It works. Thanks.